docs/topics/api/addons.rst

```diff
@@ -113,7 +113,7 @@ for autocomplete though, there are a couple key differences:
     :query string author: Filter by exact (listed) author username. Multiple author names can be specified, separated by comma(s), in which case add-ons with at least one matching author are returned.
     :query string category: Filter by :ref:`category slug <category-list>`. ``app`` and ``type`` parameters need to be set, otherwise this parameter is ignored.
     :query string lang: Activate translations in the specific language for that query. (See :ref:`translated fields <api-overview-translations>`)
-    :query string tag: Filter by exact tag name. Multiple tag names can be specified, separated by comma(s).
+    :query string tag: Filter by exact tag name. Multiple tag names can be specified, separated by comma(s).  See :ref:`available tags <tag-list>`
```

Comment:
The description for the `search` endpoint includes ", in which case add-ons containing *all* specified tags are returned.", but this one doesn't specify that. Is the behaviour in fact different, or should that be added to this description?

Reply:
Not sure, I can investigate (I didn't touch anything to do with filters or the search views, I just thought it helpful to link to the new tag page while I was at it)

Reply:
I poked around in the code and the autocomplete endpoint actually supports the same parameters as the full search endpoint (apart from `sort`, in v4+) despite the docs not mentioning them all.  I'm not going to update everything (that's veering off track for this pr) but I'll update the `tags` entry.